### PR TITLE
Formatted DevInfo.md and made the rules more clear.

### PR DESCRIPTION
**Pull request by ashutosh97**

### CODE REVIEW

File: DevInfo.md [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
+---
+
+## 2048 - HOW TO PLAY?
+ + The objective of the game is to get the number **__2048__** using additions of the number two and its multiples.
+ + You will have a grid of 16 tiles. Two numbers will be given: usually two number twos, maybe number four.
+ + Move up or down, left or right trying to join two equal numbers.
+ + When two equal numbers are in touch, they will add up.
+ + If we are run out of equal numbers on our grid, or we can not put them in touch, the **__game__** will provide us another two, a four or even an eight so we can keep on playing. If there are no free tiles on our grid, the game ends.
+ + By adding numbers, we get higher numbers and we can approach to **__2048__**, which is the goal of the game.
+ ---
```
Comment (hackrush01) [inline DevInfo.md]: Remove the three dashes ---

**Comment by hackrush01** [comment]
At the end of the commit description add this
`Fixes #2`

**Comment by ashutosh97** [comment]
Done.

**Comment by hackrush01** [comment]
Merge the commits to 1, that way it is cleaner and a good practice to follow in open source world

**Comment by ashutosh97** [comment]
Okay!

**Pull request merged**